hydra-node/json-schemas/api.yaml

```diff
@@ -1636,7 +1651,6 @@ components:
       required:
       - address
       - value
```

Comment:
We should explain in the description the semantics we apply here:

* when `witness` is present, we treat it as script output
* when `inlineDatum` is present, we don't require `witness.datum` (it's ignored)

Reply:
I added this and also tackled the `ScriptInfo` description to explain that when `datum` field is not there we are expecting inline datum.